Thanks for doing this! I'll get to finishing off these remaining ones.
```
./bot.py:2:18: W291 trailing whitespace
./bot.py:32:1: E266 too many leading '#' for block comment
./data/import_data.py:[14](https://github.com/ntjandra/RPG-bot/actions/runs/5936881890/job/16118175926?pr=3#step:5:15):19: E231 missing whitespace after ','
./data/import_data.py:19:1: E305 expected 2 blank lines after class or function definition, found 1
./data/import_data.py:29:14: E211 whitespace before '('
./data/import_data.py:29:30: E231 missing whitespace after ','
./data/import_data.py:30:46: E251 unexpected spaces around keyword / parameter equals
./data/import_data.py:30:48: E251 unexpected spaces around keyword / parameter equals
./data/import_data.py:36:1: E302 expected 2 blank lines, found 1
./data/import_data.py:39:14: E211 whitespace before '('
./data/import_data.py:39:30: E231 missing whitespace after ','
./data/import_data.py:40:57: E251 unexpected spaces around keyword / parameter equals
./data/import_data.py:40:59: E251 unexpected spaces around keyword / parameter equals
./data/import_data.py:46:1: E302 expected 2 blank lines, found 1
```

--- a/bot.py
+++ b/bot.py
@@ -1,5 +1,5 @@
 """
-Main RBG BOT File 
+Main RBG BOT File
 """
 import os
 import discord
@@ -29,7 +29,8 @@ async def on_ready():
 client.remove_command('help')
 
 
-### Command List ###~
+""" Command List """
+
 @client.command()
 async def help(ctx, *argument):
     """

--- a/bot.py
+++ b/bot.py
@@ -29,8 +29,6 @@ async def on_ready():
 client.remove_command('help')
 
 
-""" Command List """
-
 @client.command()
 async def help(ctx, *argument):
     """

--- a/data/import_data.py
+++ b/data/import_data.py
@@ -15,41 +15,43 @@ def init_data(file):
         data = json.load(f)
     return data
 
-# Init dicts to store rpg sheet data.
-characters = init_data('data/characters.json')
-skills = init_data('data/skills.json')
-
-
 def init_party():
     """
     Overwrite previous quicksaves in /data.
     """
     save_file = 'data/characters.json'
 
-    with open (save_file, "w",encoding='utf-8') as outfile:
-        json.dump(characters, outfile, indent = 4)
+    with open(save_file, "w", encoding='utf-8') as outfile:
+        json.dump(characters, outfile, indent=4)
 
     log = 'All characters have been saved and updated.'
     return log
 
-# Converts existing characters dict into a file as a JSON
 def save_player(character, slot=1):
-    # Stores existing (local) dict into a temp file
+    """
+    Converts existing characters dict into a file as a JSON.
+    Stores existing (local) dict into a temp file
+    """
     save_file = f'saves/{character}-{slot}.json'
-    with open (save_file, "w",encoding='utf-8') as outfile:
-        json.dump(characters[character], outfile, indent = 4)
+    with open(save_file, "w", encoding='utf-8') as outfile:
+        json.dump(characters[character], outfile, indent=4)
 
     log = f'Saved! Load with `!load {character} {slot}.`'
     return log
 
-# Loads a JSON formatted Sheet into /data and reinits it.
 def load_player(character, slot=1):
+    """
+    Loads a JSON formatted Sheet into /data and reinits it.
+    """
     file = f'saves/{character}-{slot}.json'
     with open(file, encoding='utf-8') as f:
         data = json.load(f)
 
-    # Map through the data to fill the characters.json values
     characters[character] = data
-
     log = f'{character} has been loaded from save slot {slot}.'
     return log
+
+# Init dicts to store rpg sheet data.
+characters = init_data('data/characters.json')
+skills = init_data('data/skills.json')
+

--- a/data/import_data.py
+++ b/data/import_data.py
@@ -11,9 +11,10 @@ def init_data(file):
     Init all relevant files
     Open JSON and returns dictionary
     """
-    with open(file,encoding='utf-8') as f:
+    with open(file, encoding='utf-8') as f:
         data = json.load(f)
     return data
+
 
 def init_party():
     """
@@ -27,6 +28,7 @@ def init_party():
     log = 'All characters have been saved and updated.'
     return log
 
+
 def save_player(character, slot=1):
     """
     Converts existing characters dict into a file as a JSON.
@@ -38,6 +40,7 @@ def save_player(character, slot=1):
 
     log = f'Saved! Load with `!load {character} {slot}.`'
     return log
+
 
 def load_player(character, slot=1):
     """
@@ -51,7 +54,7 @@ def load_player(character, slot=1):
     log = f'{character} has been loaded from save slot {slot}.'
     return log
 
+
 # Init dicts to store rpg sheet data.
 characters = init_data('data/characters.json')
 skills = init_data('data/skills.json')
-

